rcl/test/rcl/test_security_directory.cpp

```diff
@@ -34,9 +34,32 @@ static int putenv_wrapper(const char * env_var)
 #endif
 }
 
+static int unsetenv_wrapper(const char * var_name)
+{
+#ifdef _WIN32
+  // On windows, putenv("VAR=") deletes VAR from environment
+  std::string var(var_name);
+  var += "=";
+  return _putenv(var.c_str());
+#else
+  return unsetenv(var_name);
+#endif
+}
+
+class test_rcl_get_secure_root : public ::testing::Test
```

Comment:
nitpick: Make the class name CamelCase (e.g. `TestGetSecureRoot`), similar to our other tests that use fixtures. And update the test macros accordingly